### PR TITLE
feat: timeline init

### PR DESCRIPTION
**Pull request by VishnuSanal**

### CODE REVIEW

**Comment by VishnuSanal** [comment]
preview: https://vishnusanal.github.io/kochifoss.github.io/

File: themes/timeline/i18n/pt-br.toml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Translations for Portuguese
+# https://gohugo.io/content-management/multilingual/#translation-of-strings
+
+# Generic
+#
+[translations]
```
Comment (subins2000) [inline themes/timeline/i18n/pt-br.toml]: en, ml മതി. ബാക്കി ഭാഷയൊന്നും വേണ്ടാ

File: .hugo_build.lock
Comment (subins2000) [inline .hugo_build.lock:1]: This file should be gitignored

**Comment by RiyaMathew-11** [comment]
@VishnuSanal - maybe let's keep the homepage short with the highlight of the upcoming meetup, and bring the timeline to another page? - we do have to fill it up with all the meetups since now too.

File: resources/_gen/assets/scss/main.scss_8f107e2e3689beeb47e911c025a030db.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{"Target":"main.a35ab810654226b5a9aef700df8a3758ebfc89c4c14ab5186eff55e2c08ae43e.css","MediaType":"text/css","Data":{"Integrity":"sha256-o1q4EGVCJrWprvcA34o3WOv8icTBSrUYbv9V4sCK5D4="}}
```
Comment (subins2000) [inline resources/_gen/assets/scss/main.scss_8f107e2e3689beeb47e911c025a030db.json]: I believe this should be gitignored too

File: themes/timeline/README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# hugo-theme-timeline
```
Comment (subins2000) [inline themes/timeline/README.md:1]: I see there are a lot of things that come with the theme, many css, js, fonts, language files. Can we keep the current homepage https://kochifoss.org as it is and only extract just the timeline?

i.e a new theme that keeps the current styling and just extracted timeline of the timeline theme

**Review by subins2000 — CHANGES_REQUESTED** [review]
@VishnuSanal

**Comment by VishnuSanal** [comment]
superseded by #7

**Pull request closed without merging**